crates/vm/src/alu.rs

```diff
@@ -73,7 +73,7 @@ mod tests {
             Stack::Push(42).into(),
             Pred::Eq.into(),
         ];
-        eval_ops(ops, *test_access(), &EmptyState).unwrap();
+        eval_ops(ops, test_access().clone(), &EmptyState).unwrap();
```

Comment:
Just out of curiosity, why the need for all these new `.clone()`'s?

Reply:
Nice question, `struct Access` used to be `Copy` but now it's `Clone`, so the implicit copies turned into explicit clonings